planner/aamwriter/src/main/resources/examples/atos_case_study_aam.yml

```diff
@@ -73,7 +73,7 @@ groups:
         benchmark_platform: hp_cloud_services.2xl
     - dependencies:
         operation_webservices: '2'
-    - QoSRequirements:
+    - AppQoSRequirements:

```

Comment:
@rosogon , 
I see that this line is the only difference with respect the previous AAM. Can you provide us with the example of the aam generated now? The one with the requirements part changed. 

It's to use it as input in the Planner component and its modules.